### PR TITLE
run: hook server which can parse `DNSbackend` and `listen ips`

### PR DESCRIPTION
**Pull request by flouthoc**
Implement `aardvark-dns --file <config> run` to actually parse and run server

### CODE REVIEW

**Comment by flouthoc** [comment]
@mheon Merging this one as well since its a synthetic change.

File: src/commands/run.rs [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
 
 impl Run {
     /// The run command runs the aardvark-dns server with the given configuration.
     pub fn new() -> Self {
-        Self {
-        }
+        Self {}
     }
 
     pub fn exec(&self, input_file: String) -> Result<(), Error> {
```
Comment (mheon) [inline src/commands/run.rs:16]: It's not a file, it's a directory. We accept a context directory containing 1 or more config files.

Comment (flouthoc) [reply src/commands/run.rs:16]: oh I'll rename the variable in the next PR. However Its treated as a directory so it should not break.

Comment (flouthoc) [reply src/commands/run.rs:16]: @mheon Fixed in a commit specified here: https://github.com/containers/aardvark-dns/pull/6

**Pull request merged**